packages/devtools_app/lib/src/shared/side_panel.dart

```diff
@@ -13,11 +13,9 @@ import 'package:http/http.dart' as http;
 import 'package:logging/logging.dart';
 
 import '../../devtools.dart' as devtools;
-import 'common_widgets.dart';
+import '../../devtools_app.dart';
```

Comment:
do not import devtools_app.dart outside of tests. We should import libraries directly instead of importing devtools_app.dart which exports a bunch of files.